modules/migrations/gitea.go

```diff
@@ -79,12 +80,22 @@ func (g *GiteaLocalUploader) CreateRepo(repo *base.Repository, opts base.Migrate
 		return err
 	}
 
+	var remoteAddr = repo.CloneURL
+	if len(opts.AuthUsername) > 0 {
+		u, err := url.Parse(repo.CloneURL)
```

Comment:
I think this should only be allowed if the schema is `https`. See [https://golang.org/pkg/net/url/#UserPassword](https://golang.org/pkg/net/url/#UserPassword).

Reply:
Maybe we should show a warn message on ui if he is using a `http` but not `https`.